ReadingRainbow/Models/Page.swift

```diff
@@ -8,8 +8,9 @@
 import Foundation
 
 struct Page {
-    var image: Data
-    var pageNumber: Int // maybe not needed?
-    var uniqueWords: [String] // computed from paragraphs?
-    var paragraphs: [String]
+    var image: Data?
+    var pageNumber: Int? // maybe not needed?
+    var uniqueWords: [String]? // computed from paragraphs?
+    var paragraphs: [String]?
+    var content: [String]?
```

Comment:
I don't think this change should be necessary

Reply:
I also don't think we'll need `content` since that should be covered by `uniqueWords` and `paragraphs`